dubbo/src/main/java/io/seata/samples/dubbo/ApplicationKeeper.java

```diff
@@ -70,7 +70,7 @@ public void keep() {
         synchronized (LOCK) {
```

Comment:
@bigcai so，`synchronized (LOCK)` should also removed?

Reply:
Right, should remove synchronized (LOCK) in ApplicationKeeper.keep() ，LOCK.lock()  instead , and fix LOCK.lock() to avoid IllegalMonitorStateException when lock() throw exception.

'Lock.lock () should be placed before the try'
 according to 
https://stackoverflow.com/questions/10868423/lock-lock-before-try or 《阿里巴巴JAVA开发手册》